### PR TITLE
feat: Create self-introduction

### PR DESCRIPTION
**Pull request by kimdaehee90**
Leave a comment
This is my self-introduction

### CODE REVIEW

File: self-introduction.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+자기소개서
+
+<!DOCTYPE html>
```
Comment (burnyst) [inline self-introduction.html:3]: good job

**Review by burnyst — CHANGES_REQUESTED** [review]

File: self-introduction.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>self-introduction</title>
+</head>
+<body>
+    <h1>자기소개서</h1>
+    <p>
+        안녕하세요. 김대희입니다.
+        서울에 거주중이며 개발쪽 공부는 처음입니다.
+        대화하는것을 좋아하여 커피와 술도 좋아합니다.
```
Comment (burnyst) [inline self-introduction.html:14]: great

**Review by burnyst — APPROVED** [review]

File: self-introduction.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+자기소개서
```
Comment (burnyst) [inline self-introduction.html]: This is unnecessary

**Review by burnyst — CHANGES_REQUESTED** [review]

File: self-introduction.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <meta charset="UTF-8">
+    <title>self-introduction</title>
+</head>
+<body>
+    <h1>자기소개서</h1>
+    <p>
+        안녕하세요. 김대희입니다.
+        서울에 거주중이며 개발쪽 공부는 처음입니다.
+        대화하는것을 좋아하여 커피와 술도 좋아합니다.
+        긴 시간동안 함께해야 하는데 잘 지냈으면 좋겠습니다.
```
Comment (burnyst) [inline self-introduction.html:13]: great!

File: self-introduction.html (hunk 3 above)
Comment (kimdaehee90) [reply self-introduction.html]: ok

File: self-introduction.html (hunk 4 above)
Comment (kimdaehee90) [reply self-introduction.html:13]: thanks

**Review by burnyst — APPROVED** [review]
great!

**Pull request merged**